exercises/resistor-color-trio/description.md

```diff
@@ -0,0 +1,43 @@
+If you want to build something using a Raspberry Pi, you'll probably use _resistors_. For this exercise, you need to know only three things about them:
+
+- Each resistor has a resistance value.
+- Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
+  To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values.
+- Each band acts as a digit of a number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
+  In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take 3 colors as input, and outputs the correct value, in Ohms.
+  The color bands are encoded as follows:
+
+* Black: 0
+* Brown: 1
+* Red: 2
+* Orange: 3
+* Yellow: 4
+* Green: 5
+* Blue: 6
+* Violet: 7
+* Grey: 8
+* White: 9
+
+In `resistor-color duo` you decoded the first two colors. For instance: orange-orange got the main value `33`.
+The third color stands for how many zeros need to be added to the main value. The main value plus the zeros gives us a value in Ohms.
+For the exercise it doesn't matter what Ohms really are.
+For example:
+
+- orange-orange-black would be 33 and no zeros, which becomes 33 Ohms.
+- orange-orange-red would be 33 and 2 zeros, which becomes 3300 Ohms.
+- orange-orange-orange would be 33 and 3 zeros, which becomes 33000 Ohms.
+
+(If Math is your thing, you may want to think of the zeros as exponents of 10. If Math is not your thing, go with the zeros. It really is the same thing, just in plain English instead of Math lingo.)
+
+This exercise is about translating the colors into a sentence:
+
+> "The value of this resistor is … Ohms."
+
+So an input of `“orange”, “orange”, “black”` should return
+
+> “The value of this resistor is 33 Ohms.”
+
+The trick is that when we talk about Ohms, as soon as we get more than a thousand, we say “kilo”, in the same way that we say “kilograms” (kg) and “kilometers” (km). In the same way, we get “kilo-ohms” (kOhms).
```

Comment:
```suggestion
The trick is that when we talk about Ohms, as soon as we get more than a thousand, we say "kilo", in the same way that we say "kilograms" (kg) and "kilometers" (km). In the same way, we get "kilo-ohms" (kilo Ohms).
```

Reply:
I think reversing them here would already be helpful:

```suggestion
The trick is that when we talk about Ohms, as soon as we get more than a thousand, we say "kilo", in the same way that we say "kilometers" (km) and "kilograms" (kg). In the same way, we get "kilo-ohms" (kilo Ohms).
```

I'm not a fan of the double "in the same way".

Perhaps:

```suggestion
The trick is that when we talk about Ohms, as soon as we get more than a thousand, we say "kilo", which results in  "kilo-ohms" (kilo Ohms). In the same way when we have more than a thousand metres we say "kilometers" (km) and when we have more than a thousand grams, we say "kilograms" (kg).
```

Reply:
I see your point, @SleeplessByte, but I'm not convinced by this solution. I'll think of something else.

Reply:
Wouldn’t it be “kilohms” not “Kilo-ohms”? Wolfram Alpha seems think so, and that led me to NIST and https://www.nist.gov/physical-measurement-laboratory/nist-guide-si-chapter-6#623, which indicates prefixes are inseparable from units. Elsewhere it mentions lowercase presentation and that:

> there are three cases in which the final vowel of an SI prefix is commonly omitted: megohm (not megaohm), kilohm (not kiloohm), and hectare (not hectoare)

Although I’ll be honest the only one of those I’m totally sure of is hectare.

Reply:
Yes, hectare is the one I'm also sure about. @F3PiX do as you see fit ❤️ !